packages/embark/src/cmd/cmd_controller.js

```diff
@@ -111,14 +111,15 @@ class EmbarkController {
         });
       },
       function (callback) {
+        engine.startService("embarkListener");
```

Comment:
Why was this switched around?

Reply:
Great question. The `EmbarkListener` is responsible for listening to any logging output of embark. Because the `EmbarkListener` was started later in the lifecycle, there were a few logs that were not being captured, and thus the cockpit log output looked a little bit incomplete. I tried to move this above `embark.init`, but it wasn't happy. You'll notice the only missing logs in the cockpit now are the `Welcome to embark` at the top.

I will move the `engine.startService("embarkListener");` to inside the `embark.init` callback to try to get all of the missing logs included.